sbt mysubproject/run doesn't work

If trying to run Play dev mode on a sub project using `sbt mysubproject/run`, it fails with the following exception when you try to use the app:

```
[ERROR] [2014-10-07 13:40:03,019] [application] [New I/O worker #1] -

! Internal server error, for (GET) [/login] ->

java.util.NoSuchElementException: None.get
at scala.None$.get(Option.scala:313) ~[scala-library-2.11.2.jar:na]
at scala.None$.get(Option.scala:311) ~[scala-library-2.11.2.jar:na]
at play.PlayReloader$$anon$1.reload(PlayReloader.scala:88) ~[na:na]
at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.4.jar:2.3.4]
at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.4.jar:2.3.4]
at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.2.jar:na]
at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.2.jar:na]
at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.2.jar:na]
at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.2.jar:na]
at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.2.jar:na]
at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.2.jar:na]
at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.2.jar:na]
[WARN] [2014-10-07 13:40:03,097] [play] [New I/O worker #1] - No application found at invoker init
[INFO] [2014-10-07 13:40:03,352] [akka.event.slf4j.Slf4jLogger] [play-akka.actor.default-dispatcher-2] - Slf4jLogger started
```

It seems that we're not executing the `playReload` task for the right project, it's probably being run for the root project, not the selected sub project.

--- a/framework/src/sbt-plugin/src/main/scala/PlayRun.scala
+++ b/framework/src/sbt-plugin/src/main/scala/PlayRun.scala
@@ -57,12 +57,13 @@ trait PlayRun extends PlayInternalKeys {
     val args = Def.spaceDelimited().parsed
 
     val state = Keys.state.value
+    val scope = resolvedScoped.value.scope
     val interaction = playInteractionMode.value
 
     val reloadCompile = () => PlayReload.compile(
-      () => Project.runTask(playReload, state).map(_._2).get,
-      () => Project.runTask(reloaderClasspath, state).map(_._2).get,
-      () => Project.runTask(streamsManager, state).map(_._2).get.toEither.right.toOption
+      () => Project.runTask(playReload in scope, state).map(_._2).get,
+      () => Project.runTask(reloaderClasspath in scope, state).map(_._2).get,
+      () => Project.runTask(streamsManager in scope, state).map(_._2).get.toEither.right.toOption
     )
 
     val runSbtTask: String => AnyRef = (task: String) => {

